From f430434f7b73ef8d043bc4ced5f3c65670d61731 Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Wed, 09 Apr 2025 19:20:58 +0300
Subject: [PATCH] [FSW-14708] fs1: remove arch specific definitions

Removed max die count macro definition from
common header, it will be taken care from specs
arch specific defines.

Change-Id: I6167f81ea5395167784dc5b20ce4b58d1a21dd5f
---

diff --git a/common/boot_desc.h b/common/boot_desc.h
index aa4a91f..0d3cfae 100644
--- a/common/boot_desc.h
+++ b/common/boot_desc.h
@@ -17,8 +17,6 @@
 
 #define DISABLE_SLAVE_FAKE_FUSE		255
 
-#define MAX_DIE_COUNT		(FS1_NUM_IO_DIES + FS1_NUM_COMPUTE_DIES)
-
 struct boot_desc {
 	uint32_t desc_ver;
 	uint32_t spi_offset[2];
